lib/internal/bootstrap/node.js

```diff
@@ -128,6 +129,32 @@
       setupGlobalConsole();
       setupGlobalURL();
       setupGlobalEncoding();
+
+
+      let errorCodes;
+      // declared separately for name, arrow function to prevent construction
+      const queueMicrotask = (callback) => {
+        if (typeof callback !== 'function') {
+          if (errorCodes === undefined) {
+            errorCodes = NativeModule.require('internal/errors').codes;
+          }
+          throw new errorCodes.ERR_INVALID_ARG_TYPE(
+            'callback', 'function', callback);
+        }
+        _enqueueMicrotask(() => {
+          try {
+            callback();
+          } catch (e) {
+            process.emit('error', e);
+          }
+        });
+      };
+      Object.defineProperty(global, 'queueMicrotask', {
```

Comment:
Adding a new global is inherently semver-major. Adding this to `util` first would allow it to go in as semver-minor (which is the same pattern we've used for introducing other bits e.g `URL`, `TextEncoder`, `TextDecoder`, etc)

Reply:
I'm fine with semver-major.

Reply:
i'm not sure that it is inherently a semver-major in node; what code would be broken by this?

Reply:
Adding globals is semver-major as it changes the behavior of non-strict code :/.

Reply:
Only non-strict code that's omitting `var`/`let`/`const` *and* using this variable name, though - how likely is that combination?

Reply:
The policy isn't about *likely*, it's about *possible*. A case can be made to the @nodejs/tsc to request a downgrade to semver-minor.